src/main/java/countdownlatch/B_przykładCountdownLatch/Pasażer.java

```diff
@@ -0,0 +1,42 @@
+package countdownlatch.B_przykładCountdownLatch;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Zadaniem każdego pasażera jest wsiąść do pociągu.
+ *
+ * @author Kacper Staszek
+ * @author Marcin Ogorzałek
+ */
+class Pasażer implements Runnable{
+    private final Pociąg pociąg;
+    private final CountDownLatch latch;
+
+    Pasażer(Pociąg pociąg, CountDownLatch latch) {
```

Comment:
Jeśli pociąg, to zatrzask.

EDYCJA: https://duckduckgo.com/?q=latch&t=canonical&atb=v163-1&iax=images&ia=images